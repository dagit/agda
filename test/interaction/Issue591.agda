module Issue591 where

import Issue591.M
