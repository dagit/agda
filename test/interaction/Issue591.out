Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue591 (Issue591.agda).\n" t)
(agda2-info-action "*Type-checking*" " Checking Issue591.M (Issue591/M.agda).\n" t)
(agda2-info-action "*Type-checking*" " Finished Issue591.M.\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue591.\n" t)
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
Agda2> 